### PR TITLE
Feature/pdct 1597 update search result list item familylistitem to include the

### PR DESCRIPTION
**Pull request by odrakes-cpr**
# What's changed

Update Metadata tiles on the search page to include MCF corporas.  

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

### CODE REVIEW

File: src/helpers/getCategoryName.ts [hunk 1]
```diff
@@ -17,6 +17,9 @@ export const getCategoryName = (category: TCategory) => {
     case "UNFCCC":
       name = "UNFCCC";
       break;
+    case "MCF":
+      name = subCategory ? subCategory.toUpperCase() : "MCF";
```
Comment (PatFawbertMills) [inline src/helpers/getCategoryName.ts:21]: One to catch up with Conor/Dominyka later but I have a feeling they will want the full name. Can easily circle back later though

**Review by PatFawbertMills — APPROVED** [review]
LGTM

File: src/helpers/getCategoryName.ts (hunk 1 above)
Comment (odrakes-cpr) [reply src/helpers/getCategoryName.ts:21]: I will confirm in application

File: src/helpers/getCategoryName.ts [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
-    case "Executive":
-    case "Policy":
-      name = "Policy";
-      break;
-    case "UNFCCC":
-      name = "UNFCCC";
-      break;
-  }
-  return name;
+const subCategories: Record<TCorpusTypeSubCategory, string> = {
```
Comment (PatFawbertMills) [inline src/helpers/getCategoryName.ts:3]: nice

**Pull request merged**